spack/repo/packages/py-pyprecice/package.py

```diff
@@ -14,11 +14,12 @@ class PyPyprecice(PythonPackage):
 
     homepage = "https://precice.org"
     git = "https://github.com/precice/python-bindings.git"
-    url = "https://github.com/precice/python-bindings/archive/v2.0.0.1.tar.gz"
+    url = "https://github.com/precice/python-bindings/archive/v2.4.0.0.tar.gz"
     maintainers = ["ajaust", "BenjaminRodenberg", "IshaanDesai"]
 
     # Always prefer final version of release candidate
     version("develop", branch="develop")
+    version('2.4.0.0', sha256='8877da85df97d66966892440c7e777b667d8e6a33747304ed3762f4c33b729ec')
```

Comment:
One potential issue here: Old bindings versions always started with a `.1` version number. That means I would have expected a `2.4.0.1` release here.

Reply:
Yes older bindings versions did start with `.1`, but that was just wrong, so now we have moved to starting the versions with `.0`